plenum/server/node.py

```diff
@@ -132,6 +133,8 @@ def __init__(self,
         self.basedirpath = basedirpath or config.baseDir
         self.dataDir = self.config.nodeDataDir or "data/nodes"
 
+        self._view_change_timeout = self.config.VIEW_CHANGE_TIMEOUT
```

Comment:
We were thinking about a need for such timeout during initial primary selection (after a nodes are up). But looks like it's not required for now (if a primary is malicious, then a view change will be called, and a new Primary will be selected)